### DISCUSSION TITLE
Suggestion required to implement SSL/TLS termination not at LB but at Backend level

### DISCUSSION THREAD

**Discussion by mahendra03** [nginx/kubernetes-ingress#4915] [Q&A]
Hi Team,

We use nginxinc controller and as of now SSL/TLS termination happen at LoadBalancer but we want this termination happen at Backend.

a) Wanted to know what all changes is required in ingress controller level?
b) wanted to know what all changes will be required by application team.

**Comment by vepatel** [comment]
closing this one as the issue is being discussed in https://github.com/nginxinc/kubernetes-ingress/discussions/4916
